<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_8">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1401</width>
      <height>901</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background-color:rgb(30,31,34)
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>961</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>1071</width>
       <height>401</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
height:120px;
background-color:rgb(160,160,160);
border-radius:25px;
border:none;
color:rgb(255,255,255);
font-size:50px;
font-style:none;
font: 57 35pt &quot;Adobe Myungjo Std M&quot;;
}
QPushButton::hover{
background-color:rgb(65,200,255);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <widget class="QFrame" name="frame_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>139</y>
        <width>1071</width>
        <height>261</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_14">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_option_1">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>选项1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_option_3">
            <property name="text">
             <string>选项3</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_15">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_option_2">
            <property name="text">
             <string>选项2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_option_4">
            <property name="text">
             <string>选项4</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_problem">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>571</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size:30px;
background-color:rgb(30,31,34);
text-align:center;
border-radius:20px;
color:rgb(255,255,255);
font: 57 36pt &quot;Adobe Myungjo Std M&quot;;
}</string>
      </property>
      <property name="text">
       <string>所抽查的单词英文</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>480</y>
       <width>541</width>
       <height>231</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTextBrowser" name="textBrowser_translation">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>531</width>
        <height>211</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextBrowser{
font-size:50px;
background-color:rgb(160,160,160);
border-radius:25px;
color:rgb(255,255,255);
font: 25 40pt &quot;Adobe 宋体 Std L&quot;;
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_13">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>490</y>
       <width>531</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
height:120px;
background-color:rgb(160,160,160);
border-radius:25px;
border:none;
font-size:50px;
font-style:none;
color:rgb(255,255,255);
font: 25 &quot;Adobe 宋体 Std L&quot;;
}
QPushButton::hover{
background-color:rgb(70,203,255)
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_translation">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>531</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>查看中文意思</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_next">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>531</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>下一题</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
